data/human/jobs.txt

```diff
@@ -2730,6 +2730,7 @@ mission "Bounty Hunting (Very Tiny)"
 			"combat rating" > 15
 			has "ships: Interceptor"
 			has "ships: Light Warship"
+			has "ships: Light Carrier"
```

Comment:
Light Carrier arguably is parallel to Medium Warship rather than Light Warship, so I'd say it shouldn't be included here.